File: test/dotnet/ExchangeInfo.cs
using System;

using Mexc.Sdk;
using Newtonsoft.Json;
namespace dotnet
{
   public class ExchangeInfo
   {
       public static void Main(string[] args)
       {
            var spot = new Spot();

            Dictionary<string, object> options = new Dictionary<string, object>();
            options.Add("symbol", "BTCUSDT");

            var exchangeInfoRes = spot.ExchangeInfo(options);
            Console.WriteLine("get exchange info");
            Console.WriteLine(JsonConvert.SerializeObject(exchangeInfoRes));
       }

   }
}

File: test/java/ExchangeInfo.java
import Mexc.Sdk.*;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

public class ExchangeInfo {

    public static void main(String[] args) {       
        Spot client = new Spot();

        HashMap options = new HashMap();
        options.put("symbol","BTCUSDT");

        System.out.printf("ExchangeInfo" + client.exchangeInfo(options));
    }
}